code/fundamentals.rb

```diff
@@ -8,8 +8,18 @@
 
 # 1. Create a variable `my_favorite_animal` and assign it the name of your favorite animal
 
+my_favorite_animal = "panda"
+
 # 2. Print "My favorite animal is <your-favorite-animal>" by printing a string with `my_favorite_animal` interpolated into it.
 
+puts "My favorite animal is #{my_favorite_animal}."
+
 # 3. Write a method `make_noise` which prints the string "BANG!"
 
+def make_noise
+  "BANG!"
+end
+
+puts make_noise
```

Comment:
👍 note that the phrasing "method... which prints" implies the method should do the priniting